src/main/java/uk/gov/hmcts/cmc/claimstore/events/response/DefendantResponseCitizenNotificationsHandler.java

```diff
@@ -51,4 +63,19 @@ public void notifyClaimantResponse(DefendantResponseEvent event) {
             referenceForClaimant(claim.getReferenceNumber())
         );
     }
+
+    @EventListener
+    public void uploadDocument(DefendantResponseEvent event) {
```

Comment:
This method does not belong in this class. This is a citizen notifications handler.

Reply:
There is no clear indication as to where this method must ideally belong. Can you please elaborate why it should not be here ?

Reply:
The class name `DefendantResponseCitizenNotificationsHandler` suggests its a notification handler for citizens. But this method has nothing to do with citizen notification. This method is generating a document and then firing an event for document upload.